apps/aehttp/src/sc_ws_api_jsonrpc.erl

```diff
@@ -234,6 +234,9 @@ process_incoming(Msg, FsmPid) ->
     end.
 
 -spec process_request(map(), pid()) -> no_reply | {reply, map()} | {error, term()}.
+process_request(#{<<"method">> := <<"channels.system">>,
```

Comment:
It seems requires doc at https://github.com/aeternity/protocol/blob/8b2724d14d8b4878b6b2c74850ec3f7f7716aeb2/node/api/channel_ws_api.md ?